redbeams/src/main/java/com/sequenceiq/redbeams/domain/DatabaseConfig.java

```diff
@@ -0,0 +1,205 @@
+package com.sequenceiq.redbeams.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.DatabaseVendor;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.ResourceStatus;
+import com.sequenceiq.cloudbreak.aspect.secret.SecretValue;
+import com.sequenceiq.cloudbreak.auth.altus.Crn;
+import com.sequenceiq.cloudbreak.domain.Secret;
+import com.sequenceiq.cloudbreak.domain.SecretToString;
+import com.sequenceiq.redbeams.converter.database.CrnConverter;
+
+@Entity
+public class DatabaseConfig {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "databaseconfig_generator")
+    @SequenceGenerator(name = "databaseconfig_generator", sequenceName = "databaseconfig_id_seq", allocationSize = 1)
+    private Long id;
+
+    @Convert(converter = CrnConverter.class)
+    @Column(nullable = false)
+    private Crn crn;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(length = 1000000, columnDefinition = "TEXT")
+    private String description;
+
+    @Column(nullable = false, name = "connectionurl")
+    private String connectionURL;
+
+    @Column(nullable = false, name = "database_engine")
+    @Enumerated(EnumType.STRING)
+    private DatabaseVendor databaseEngine;
```

Comment:
This is perfectly fine for this PR. I opted to name this field "database vendor" in my stuff, so we should sync up and pick whether to use "engine" or "vendor", and change one or the other in a later review.

Reply:
Again a copy paste thing ... For me both is ok, so I change it to vendor.